### PR TITLE
Grammer and language

### PR DESCRIPTION
**Pull request by ArshErgon**
Corrected some grammar in Readme.md, CONTRIBUTING.md.

### CODE REVIEW

File: runtime.txt [hunk 1]
```diff
@@ -1 +1 @@
-3.8
+python 3.8
```
Comment (hudeven) [inline runtime.txt:1]: @ArshErgon I think Netlify requires this file only containing a version number: https://docs.netlify.com/configure-builds/manage-dependencies/#python

**Review by hudeven — CHANGES_REQUESTED** [review]
Overall LGTM. @ArshErgon thanks for improving the documentation! I will approve after you sign the CLA and resolve the comment.

File: runtime.txt (hunk 1 above)
Comment (ArshErgon) [reply runtime.txt:1]: @hudeven Sorry it's my first time, I have updated the readme.txt

**Review by hudeven — APPROVED** [review]
LGTM. @ArshErgon thanks for improving the documentations!

**Pull request merged**